plugins/cube/cube.cpp

```diff
@@ -479,6 +467,19 @@ class wayfire_cube : public wayfire_plugin_t
             0.0f, 0.0f
         };
 
+        update_workspace_streams();
```

Comment:
actually if you're gonna move things, the `vertexData` should be declared right before uploading it, also make it static please.

Reply:
This should be better now.